engine/release-notes/index.md

```diff
@@ -13,896 +13,164 @@ redirect_from:
 This document describes the latest changes, additions, known issues, and fixes
 for Docker Engine.
 
-# Version 20.10
+## 23.0.0
```

Comment:
Been thinking about this for some time, but never started working on that.

First; we should consider doing a _rename_ of `index.md` to `20.10.md` (and fixing the links to the page) in a separate commit. That commit should _not_ add a new `index.md`, only a rename. That way, `git` will properly consider it a "rename", and preserve git history on the `20.10.md` file.

Second (this is the thing I wanted to work on, but didn't get to): we should use a similar approach as we take for the API reference, and make `index.md` (or have a `latest.md`) _redirect_ to the versioned release note (so currently `/engine/release-notes/` would redirect to `/engine/release-notes/20.10/`). That way we have proper "permalinks" for release notes (which may be referenced in (e.g.) CVE databases etc), but continue to have an easy-to-find URL for "what's the latest release note?".

Currently, we effectively break _all_ links to release notes when we do a new release, which isn't "great" :joy:

Reply:
argh, yes I noticed too late that I'd messed up the git records for these changes. I will fix it 😅

Reply:
regarding the latest/index redirect, we could also have the `index` file semantically represent a list page of the release notes, and always have each release note file versioned in the filename. Then we don't have to mess with redirects at all (unless we specifically want a `latest` endpoint)